.github/workflows/build.yml

```diff
@@ -104,7 +104,8 @@ jobs:
           python-version: '3.9'
           architecture: ${{ matrix.arch }}
       - run: python -m pip install pefile machomachomangler
-      - run: python utilities/wheel-fix.py ./dist/*.whl ./secrethub/CLIENT.dll ./wheelhouse
+      - run: git clone https://github.com/secrethub/python-wheel-repair.git
```

Comment:
As a reference: this is done to correctly attribute the original source of this script, which was published under a GPL license, which requires our modifications to also be published under a GPL license. To make this clear, this code has been published in a separate repo, as that there can be no confusion about the fact that `secrethub-python` can be published under the Apache 2.0 license.